source/developers/references/internal_data_formats/tracking_logs/common_fields.rst

```diff
@@ -0,0 +1,259 @@
+
+.. _common:
+
+*****************************
```

Comment:
Is there a reason this didn't start with hashes for the page title?

Reply:
Just copied in that way, fixed now